servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -249,61 +249,22 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 	};
 
 	struct SceneState {
-		// This struct is loaded into Set 1 - Binding 0, populated at start of rendering a frame, must match with shader code
+		// This struct is loaded into Set 1 - Binding 20, populated at start of rendering a frame, must match with shader code
```

Comment:
This should maybe become binding 1 with all the other bindings increasing by 1...

Reply:
made this so